src/arrays/referencing_array_and_its_elements/hard/q3/EvenorOdd.java

```diff
@@ -0,0 +1,46 @@
+package arrays.referencing_array_and_its_elements.hard.q3;
+/*
+Write a Java program that defines the array of integers with positive numbers ( [3, 4, 6, 21, 48, 42, 89, 2, 5, 91] ).
```

Comment:
This question also needs to be switched to use scanner input so that they can't hardcode based on the array we provide them.